config/prometheus/prometheus-configmap.yaml

```diff
@@ -14,7 +14,24 @@ data:
         static_configs:
           - targets: ['localhost:9090']
 
-      - job_name: 'hive'
-        static_configs:
-          - targets: ['hive-controllers:2112']
+      - job_name: 'hive-controllers'
+        kubernetes_sd_configs:
+          - role: endpoints
```

Comment:
just for next time https://prometheus.io/docs/prometheus/latest/configuration/configuration/#endpoints

Reply:
Could you clarify what you mean above? I'm not following.